### PR TITLE
#6363: observe max tensix slots in bidir tunneller

### PR DESCRIPTION
**Pull request by aliuTT**

### CODE REVIEW

**Comment by aliuTT** [comment]
Clean CI: https://github.com/tenstorrent-metal/tt-metal/actions/runs/8301184450

File: tt_metal/hw/inc/ethernet/tunneling.h [hunk 1]
```diff
@@ -333,6 +340,13 @@ FORCE_INLINE void eth_tunnel_dst_forward_one_cmd(db_cb_config_t *eth_db_cb_confi
 
     routing_info->dst_acked_valid_cmd = 0;
 
+    // Increment available issue slot on remote DST
+    if constexpr(sender_is_issue_path) {
+        *remote_issue_cmd_slots -= 1;
+    } else {
+        *remote_issue_cmd_slots += 1;
```
Comment (DrJessop) [inline tt_metal/hw/inc/ethernet/tunneling.h:347]: No need for atomic incr?

**Review by DrJessop — APPROVED** [review]
Left some comments that you can feel free to address, but approving regardless.

File: tt_metal/hw/inc/ethernet/tunneling.h (hunk 1 above)
Comment (aliuTT) [reply tt_metal/hw/inc/ethernet/tunneling.h:347]: this is local L1 so no need right?

Comment (DrJessop) [reply tt_metal/hw/inc/ethernet/tunneling.h:347]: as long as no one else is modifying this

Comment (aliuTT) [reply tt_metal/hw/inc/ethernet/tunneling.h:347]: Yea, no one else is modifying

**Pull request merged**